.travis.yml

```diff
@@ -38,6 +33,11 @@ notifications:
       - "%{repository}#%{build_number} [%{branch}: %{commit} by %{author}] %{message}"
       - "Changes: %{compare_url} | Details: %{build_url}"
 matrix:
+  include:
+  - os: linux
```

Comment:
Maybe this should have an explicit `LANG=<something else>`?

Reply:
I guess `LANG=C` might do it; I imagine that's the default. `LANG=` most closely matches the machines on which it's failing, though (they don't have the locale set at all).